src/tests/ftest/pool/destroy_tests.yaml

```diff
@@ -1,38 +1,23 @@
-# change host names to your reserved nodes, the
-# required quantity is indicated by the placeholders
 hosts:
-  test_machines1:
-    - boro-A
-  test_machines2:
-    - boro-A
-    - boro-B
-  test_machines2a:
-    - boro-C
-    - boro-D
-  test_machines3:
-    - boro-A
-    - boro-B
-    - boro-C
-  test_machines6:
+  test_servers:
     - boro-A
     - boro-B
     - boro-C
     - boro-D
     - boro-E
     - boro-F
+  test_clients:
```

Comment:
Is client node explicitly needed? As we are just using 1 client node can we simply use the avocado's default agent which it uses as client.
Normally the node from which the tests are launched from acts like the client node as well.

Reply:
I would vote for explicitly calling out the client node.   We may not always want to run the client on the host launching the test case.

Reply:
We should be moving away from running the client on the node that runs the test.  It's good testing practice to isolate the code being tested from the node running the tests as we don't want the code under test from being able tank or otherwise interfere with the test running.

Reply:
Sure that's fine with me.